torch/_inductor/graph.py

```diff
@@ -97,6 +97,15 @@ def is_magic_method(op):
     return op in magic_ops
 
 
+def view_to_reshape(gm):
+    """
+    Replace view ops in the GraphModule to reshape ops.
+    """
+    for nd in gm.graph.nodes:
+        if nd.target == torch.ops.aten.view.default:
+            nd.target = torch.ops.aten.reshape.default
```

Comment:
This might be better to put in `fx_passes/post_grad.py`

Can we just enable this one always for uniformity?

Reply:
Yea. I also realized in current version of this PR, a few models will fail in the FakeTensorProp call within compile_fx_inner for the backward graph. The reason is due to layout change for the backward graph's inputs.

It start to fail because I recently moved the call to view_to_reshape after FakeTensorProp.

We can call it uniformly and before FakeTensorProp.